### DISCUSSION TITLE
Can uploaded files be read visually

### DISCUSSION THREAD

**Discussion by zhangyang-1** [requarks/wiki#6056] [General]
In the current version, create a new page, select Insert File (such as a pdf file) and save the page. Click a file on the page. The file is downloaded to the local PC. Can I open the contents of the file and read it on the wiki? This is going to be a little bit more intuitive

**Comment by DarickBrokaw** [comment]
Had the same idea. Then I found this link, [https://developer.adobe.com/document-services/docs/overview/pdf-embed-api/](https://developer.adobe.com/document-services/docs/overview/pdf-embed-api/)
However, after trying everything on the page, I cannot get anything to work (Wiki.js 2.5.296)(ChromeVersion 109.0.5414.121)

I liked the idea of using the Adobe PDF Embed API. But, while I can create a desktop "Index.html" and past the [example code](https://developer.adobe.com/document-services/docs/overview/pdf-embed-api/) (+ my Client ID that I had to setup), and it works, I can not do it in Wiki.js in either the markdown or the Raw HTML pages. 

The most interesting thing is that it appeared for a few seconds after pasting the scripts to the scripts pages and the div to the prime HTML page, but it disappeared. Now, I can not even reproduce the appearance and disappearance of the PDF embed.

I am very frustrated, and I feel close but missing something small. I am wondering if there is not a feature in the admin section that needs to be turned on or off, like the iframe embed toggle (mine is allowed).
